api-gateway/src/main/kotlin/org/cqfn/save/gateway/security/WebSecurityConfig.kt

```diff
@@ -185,22 +183,8 @@ class WebSecurityConfig(
                     }
                 )
             )
-            .authenticated()
+            .permitAll()
```

Comment:
Maybe add some comments here too, why do we `permitAll` from now

Reply:
Yes, added